subsys/fs/Kconfig

```diff
@@ -22,12 +22,61 @@ config FILE_SYSTEM_SHELL
 	This shell provides basic browsing of the contents of the
 	file system.
 
+choice
+	prompt "Supported file system"
+	default FILE_SYSTEM_FAT
+
 config FILE_SYSTEM_FAT
 	bool "FAT file system support"
-	default y
 	help
 	Enables FAT file system support.
 
+config FILE_SYSTEM_NFFS
+	bool "NFFS file system support"
+	help
+	Enables NFFS file system support.
+
+endchoice
+
+menu "NFFS Settings"
+	visible if FILE_SYSTEM_NFFS
+
+config FILE_SYSTEM_NFFS_FLASH_DEV_NAME
```

Comment:
I suggest we rename FILE_SYSTEM_NFFS_* to FS_NFFS_*. We made this mistake in Bluetooth and it quickly becomes too long.